runtime/src/test/java/io/rx_cache/internal/InvalidUsageTest.java

```diff
@@ -0,0 +1,51 @@
+package io.rx_cache.internal;
+
+import org.junit.ClassRule;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runners.MethodSorters;
+
+import java.io.File;
+import java.security.InvalidParameterException;
+
+/**
+ * Created by daemontus on 02/09/16.
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class InvalidUsageTest {
+    @ClassRule
+    public static TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Test(expected = InvalidParameterException.class)
+    public void _00_Cache_Directory_Null() {
```

Comment:
Remove _00_ from the name method. We don't need that the methods preserve any order when they are executed.